hi-ml-azure/src/health_azure/utils.py

```diff
@@ -2020,3 +2029,21 @@ def check_is_any_of(message: str, actual: Optional[str], valid: Iterable[Optiona
     if actual not in valid:
         all_valid = ", ".join(["<None>" if v is None else v for v in valid])
         raise ValueError("{} must be one of [{}], but got: {}".format(message, all_valid, actual))
+
+
+def get_amulet_keys_not_set() -> Set[str]:
+    """
+    Check environment variables for a given set associated with Amulet jobs. Returns a set containing any keys
+    that are not available
+    """
+    print(os.environ)
```

Comment:
The print statement will run for all jobs, amulet or not, and produce a huge amount of output. Can we guard by `logging.debug`?

Reply:
This should have been deleted!